src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1177,6 +1177,10 @@ protected virtual void AssignImpl(BoundNode node, BoundExpression value, bool is
                     ((BoundTupleExpression)node).VisitAllElements((x, self) => self.Assign(x, value: null, isRef: isRef), this);
                     break;
 
+                case BoundKind.SuppressNullableWarningExpression:
```

Comment:
I assume you mean this would be a new error (I haven't seen such error before).

As a side note, I'm still working on your earlier test suggestions (permutation testing). I haven't addressed all the feedback yet.

Reply:
Yes, sorry I meant it we should an a new error. I'm not sure about `F(out x!)` though. Should that be allowed?

---
In reply to: [193450450](https://github.com/dotnet/roslyn/pull/27317#discussion_r193450450) [](ancestors = 193450450)

Reply:
I think `out x!` should be allowed, as it allows you to say: "I know this method doesn't return null in this scenario". I'll take a note to confirm with LDM though.

Reply:
Filed https://github.com/dotnet/roslyn/issues/27522